Exiled.Loader/Loader.cs

```diff
@@ -242,13 +242,17 @@ public static void ReloadPlugins()
                     plugin.Config.IsEnabled = false;
 
                     plugin.OnDisabled();
+
+                    plugin.OnUnregisteringCommands();
```

Comment:
I'd recommend unregistering commands before calling OnDisabled(), simple because OnDisable could null objects or instances of classes that the commands need to use, and if (albiet unlikely), the OnDisable() call is made, and then a plugin command is invoked before OnUnregisteringCommands() is called, it could cause a nullref etc.

It's **extremely** unlikely, but might aswell be safe than sorry.

Reply:
Fixed, also changed that in DisablePlugins() method